src/redux/actions/authUser.js

```diff
@@ -0,0 +1,33 @@
+// Signup user
+import axios from "axios";
+
+import { SIGNUP_SUCCESS, SIGNUP_ERROR } from "../action_types";
+import { serverError, startFetch } from "./common";
+
+export const signupSuccess = message => ({
+  type: SIGNUP_SUCCESS,
+  message
+});
+
+export const signupError = error => ({
```

Comment:
Add comments to both action creators. Action comments are in the format:
`/**
   * SignUp unsuccessfully
   *
   * @param {string} error
   * @return {object} type and payload
   */`

Reply:
I'll add comments.

Reply:
@PAbishai Yes the comment is good but it is on the wrong line. It's supposed to be before the action creator not within it, hence on line 5.

Reply:
@kimotho-njoki  I have refactored